### ISSUE TITLE
Generate graph visualization from database

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#80]
## User story
1. As a user
2. I want the “Generate” button to show the generated knowledge graph
3. So that I can visually see how the information of the document I uploaded is linked.

## Acceptance criteria
* When user clicks on the "generate" button after uploading a PDF, a knowledge graph will be generated from that pdf (trigger the backend generation)
* Integrate with backend API (get graph data from JanusGraph)
* Once the knowledge graph is finished generating, it should automatically be displayed in the UI

## Definition of done (DoD)
* Acceptance criteria is satisfied
* Pull request to dev branch
* Code-reviewed by peer
* Approve code and merge into dev branch
* Automated tests are run and passed
* When necessary, update software architecture diagram/documentation and bill of materials

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
